src/components/Integration.vue

```diff
@@ -98,6 +98,10 @@ export default Integration;
   }
   .integrations li {
     list-style-type: none;
-    white-space: pre-wrap;
+    white-space: pre-line;
+  }
+  ul.integrations::v-deep div.checkbox-outer-container-description {
+    margin-top: -6px;
+    margin-bottom: 15px;
```

Comment:
The `margin-bottom` would be slightly more semantic on the `li` instead, but this is probably close enough for now.

Reply:
I've filed https://github.com/rancher-sandbox/rancher-desktop/issues/649 to deal with display of the utilities.